src/utils/utils_common.h

```diff
@@ -116,8 +73,8 @@ static inline void align_ptr_size(void **ptr, size_t *size, size_t alignment) {
         s -= alignment - rest;
     }
 
-    ASSERT((p & (alignment - 1)) == 0);
-    ASSERT((s & (alignment - 1)) == 0);
+    assert((p & (alignment - 1)) == 0);
```

Comment:
Why did you replace UMF `ASSERT` with standard `assert`?

Reply:
So that utils_common.h is independent from utils_assert.h

Reply:
I understood that, but why do you want `utils_common.h` to be independent from `utils_assert.h`?

Reply:
It is needed by https://github.com/oneapi-src/unified-memory-framework/pull/320

Reply:
Sorry, but I still do not understand what is the exact issue...

Reply:
> @vinser52 we don't want to include fprintf into proxy lib because it wouldn't build. But I don't like this solution either - we shouldn't use standard assert

Now it is clear. Thank you for your explanations.  It should be fixed when we introduce a logger, right?

Reply:
right - with logger we wouldn't need to include stdlib which cause problems on windows

Reply:
So, maybe we can go with current PR, but create TODO comment and issue to track it. So that we can fix it when logger is merged.